### ISSUE TITLE
Change from PySide to PyQT 

### ISSUE THREAD

**Issue by rafaelscheel** [aaronportnoy/toolbag#11]
Hey 

Apperantly IDA uses now PyQT and not PySide anymore (http://www.hexblog.com/?p=906).

I adjusted the code a bit, so that it works with PyQt now, but get an crash on exit (hehehe). Did anybody already successfully change from PySide to PyQt? Just to make sure we don't do the work twice :P

Cheers,
Rafael

**Comment by rafaelscheel** [comment]
Fixes work for IDA on 64Bit Windows (there for IDA 32Bit and 64Bit)... Apperantly not in a 32-Bit Windows VM :/

**Comment by aaronportnoy** [comment]
@rafaelscheel I haven't tested it on IDA with PyQt yet, thanks for taking a look. I'll try to free up some time in July to make sure it works alright. 

What's the crash you see?

**Comment by rafaelscheel** [comment]
@aaronportnoy Yes I fixed it, so it runs now with PyQt (not as before PySide). I will fork the project on Friday and upload the changes (Its a no-brainer, but still many small changes). This should make it eazy to keep both versions with PySide and PyQt. 

Regarding the crash: I will document / upload the crashes as well on Friday. It works perfectly on x64 Machines (IDA x64 but also x32). It crashes on Exit on my x32 Win7 VM. I will upload the crash dump and so on as well latest on friday.

**Comment by rafaelscheel** [comment]
@aaronportnoy just made the pull request. Regarding the crash, i uploaded the ida dmp here:
https://drive.google.com/open?id=0ByQKtdCNEYS4dWFYMkFRVjdkVHM

Might not be too relevant, lets see if others have the crash too.

**Comment by kokanin** [comment]
puh-lease implement these changes, having multiple idas installed screws my workflow :D
